.eslintrc

```diff
@@ -10,7 +14,12 @@
     "newline-after-var": [0],
     "react/jsx-indent-props": [2, 2], // no tabs, indent is two spaces
     "react/jsx-key": [2], // validate that key prop exists
-    "react/jsx-no-undef": [2] // disallow undeclared variables in JSX
+    "react/jsx-no-undef": [2], // disallow undeclared variables in JSX
+    "no-unused-vars": [ 1, {
+      "vars": "local",
+      "argsIgnorePattern": "action"
+    }],
+    "comma-dangle": [0]
```

Comment:
this I'm not sure about - we had several violations in the codebase so I thought it might be something we want turned off.